pkg/kubelet/kubelet_volumes.go

```diff
@@ -128,6 +130,18 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*v1.Pod, runningPods []*kubecon
 			orphanVolumeErrors = append(orphanVolumeErrors, fmt.Errorf("Orphaned pod %q found, but volume paths are still present on disk", uid))
 			continue
 		}
+
+		// If there are any volume-subpaths, do not cleanup directories
```

Comment:
i do not fully follow why we should not cleanup directories if volume-subpaths exists ? May be add some additional comments to explain that

Reply:
It means that all the volumes have not been cleanly unmounted yet